draft-packages/form/KaizenDraft/Form/SearchField/SearchField.tsx

```diff
@@ -4,10 +4,10 @@ import searchIcon from "@kaizen/component-library/icons/search.icon.svg"
 import { Icon } from "@kaizen/component-library"
 import spinnerIcon from "@kaizen/component-library/icons/spinner.icon.svg"
 import clearIcon from "@kaizen/component-library/icons/clear-white.icon.svg"
-import { FieldMessage, Label } from "@kaizen/draft-form"
-import styles from "./styles.module.scss"
+import { FieldMessage, Label } from ".."
+import styles from "./SearchField.module.scss"
 
-export interface SearchBoxProps {
+export interface SearchFieldProps {
   id: string
   placeholder?: string
   labelText?: string | React.ReactNode
```

Comment:
`labelText` is an optional prop here, but should it be? Is there a scenario in which there would be no label?

And if there is a default label as a fallback, shouldn't that be specified on [line 35](https://github.com/cultureamp/kaizen-design-system/pull/976/files#diff-32ba8cb1fb45d7fad8385031e190a9802d23fc9581d5c8da35f2a85cb4bb6f7bR35)?

(I'm not saying there _is_ a default label – that's a design question.)

Reply:
The [documentation](https://cultureamp.design/components/search-field/) for this component has some stuff about labels. I would make `labelText` required, and we can create a `labelVisible` prop
![image](https://user-images.githubusercontent.com/1811583/106078824-2c244680-6168-11eb-9ed0-cbb7249649d0.png)